go.sum

```diff
@@ -508,6 +508,8 @@ github.com/gopherjs/gopherjs v0.0.0-20181017120253-0766667cb4d1/go.mod h1:wJfORR
 github.com/gorilla/mux v1.8.1 h1:TuBL49tXwgrFYWhqrNgrUNEY92u81SPhu7sTdzQEiWY=
 github.com/gorilla/mux v1.8.1/go.mod h1:AKf9I4AEqPTmMytcMc0KkNouC66V3BtZ4qD5fmWSiMQ=
 github.com/grpc-ecosystem/grpc-gateway v1.16.0/go.mod h1:BDjrQk3hbvj6Nolgz8mAMFbcEtjT1g+wF4CSlocrBnw=
+github.com/hallyn/umoci v0.0.0-20240122192320-3c661c1beb44 h1:4xaZ4I8PCnycSDr0RHn0JJVJtuJr7Q852ui8ChOI8rQ=
```

Comment:
Run a go mod tidy. Should update these lines and fix the compilation errors.

Reply:
Will do, but note that I had done a go mod tidy, a full build, and a full test locally yesterday.  Seems fragile...

Reply:
Done, hoping for the best, thanks :)